### PR TITLE
New codebase setup

### PR DESCRIPTION
**Pull request by TStrawberry**
## Overview
Setup new codebase for ARK-iOS

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## In scope
- Config .gitignore to ignore some noise
- Create initial Xcode project
- Config bundle & fastlane & cocoapods
- Config pull request template
- Config scripts to initialise project

## Check list
1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?
3. [x] Have you update the wiki of your change?

## Reference
- Card: https://trello.com/c/X0fOSBxc
- Related PR: https://github.com/TW-Smart-CoE/ARK-WIKI/pull/1

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  end
+
+  desc "Runs all the unit tests"
+  lane :unit_tests do
+    run_tests(scheme: "ARK-iOS")
+  end
+
+  desc "Notify message to user"
+  private_lane :notify_message do |options|
+    if is_ci
```
Comment (WeidongGu) [inline fastlane/Fastfile:42]: `is_ci` 这个是从哪里来的参数呀？

File: README.md [hunk 2]
```diff
@@ -1 +1,36 @@
...
+
+## System Requirements
+* Ruby 2.6.5
+* Bundler 2.3.19
+
+## Setup(Mac)
+- Install ruby & bundler
+  ```sh
+  brew install ruby
+  bundle install
```
Comment (pujiaxin33) [inline README.md]: Maybe should change to `gem install bundler`

File: fastlane/Fastfile (hunk 1 above)
Comment (Tony---Zhang) [reply fastlane/Fastfile:42]: 这个是Fastlane的默认参数

File: README.md (hunk 2 above)
Comment (TStrawberry) [reply README.md]: Good catch!!! 🙏

**Review by pujiaxin33 — APPROVED** [review]
项目现在的搭建目前以SwiftUI为基础，会考虑引入UIKit的东西吗？

**Review by Tony---Zhang — APPROVED** [review]

**Comment by Tony---Zhang** [comment]
> 项目现在的搭建目前以SwiftUI为基础，会考虑引入UIKit的东西吗？

我倾向于不用？

**Pull request merged**